fontir/src/glyph.rs

```diff
@@ -416,8 +417,8 @@ fn flatten_glyph(context: &Context, glyph: &Glyph) -> Result<(), BadGlyph> {
     if glyph.default_instance().components.is_empty() {
         return Ok(());
     }
-    trace!(
-        "Flatten {} {:?}",
+    log::debug!(
+        "flattening {} (components: {:?})",
```

Comment:
Nit: capitalization of the first word of the log message is inconsistent. I think mostly we capitalize so Flattening